README.md

```diff
@@ -47,13 +68,13 @@ plugins:
           400: ['*']
 ```
 
-> **Note:** If you have no `plugins` entry in your config file yet, you'll likely also want to add the `search` plugin. MkDocs enables it by default if there is no `plugins` entry set, but now you have to enable it explicitly.
+## Performance
 
-To enable cross-page anchor validation, you must set `use_directory_urls = False` in `mkdocs.yml`:
+[BeautifulSoup](https://beautiful-soup-4.readthedocs.io/en/latest/) is used to parse HTML as part of
+validation. When `lxml` and `cchardet` are installed in the Python environment, a faster parser will
```

Comment:
Let's change this now that lxml is a hard dependency.

Reply:
good call, done